nltk/internals.py

```diff
@@ -523,6 +529,12 @@ def find_file(filename, env_vars=(), searchpath=(),
     return next(find_file_iter(filename, env_vars, searchpath,
                                file_names, url, verbose))
 
+
+def find_dir(filename, env_vars=(), searchpath=(),
+        file_names=None, url=None, verbose=True):
+    return next(find_file_iter(filename, env_vars, searchpath,
+                               file_names, url, verbose, finding_dir=True))
+
```

Comment:
I'm not sure why git didn't detect the changes but it's done, see https://github.com/alvations/nltk/blob/patch-1/nltk/internals.py#L533